tools/doc/json.js

```diff
@@ -7,6 +7,15 @@ module.exports = doJSON;
 
 var marked = require('marked');
 
+// customized heading without id attribute
```

Comment:
It's all the same as the HTML output. I put diff results of all.html and all.json in the description of change.